sw/device/lib/testing/json/command.h

```diff
@@ -21,6 +21,7 @@ extern "C" {
     value(_, I2cStartTransferWriteSlow) \
     value(_, I2cStartTransferRead) \
     value(_, I2cStartTransferWriteRead) \
+    value(_, I2cTestConfiging) \
```

Comment:
```suggestion
    value(_, I2cTestConfig) \
```

Reply:
Good catch, that was a failed search and replace.